iiif_prep/iiif_prep/archivesspace.py

```diff
@@ -0,0 +1,40 @@
+from asnake.aspace import ASpace
+
+
+class NoResultsError(Exception):
+    pass
+
+
+class MultipleResultsError(Exception):
+    pass
+
+
+class ArchivesSpaceClient:
+    def __init__(self, baseurl, username, password, repository):
+        self.client = ASpace(
+            baseurl=baseurl,
+            username=username,
+            password=password,
+            repository=repository).client
+        self.repository = repository
+
+    def get_diary_refid(self, diary):
+        refid = None
+        search_url = "repositories/{}/search?page=1&type[]=archival_object&q=digital_object:+{}.pdf".format(
+            self.repository, diary)
+        results_page = self.client.get(search_url).json()
```

Comment:
Would suggest a bit of a refactor (untested)
```suggestion
        params = {"page": 1, "type[]": "archival_object", "q": "digital_object:+{}.pdf".format(diary)}
        results_page = self.client.get("repositories/{}/search".format(self.repository), params=params).json()
```
Keeps things a little more pythonic and easier to read

Reply:
I wasn't able to get that to work, even after doing a close reading of https://archivesspace.github.io/archivesspace/api/#search-this-repository

Reply:
Weird, wonder if that's an ArchivesSnake bug, or if it expects to get those params passed in a different way...